composer.json

```diff
@@ -45,6 +45,8 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "symfony/security-acl": "~2.4|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
> In your travis ci file, you're actually using the latest stable release of phpunit. Is that really what you want? All the UnitTests are written with a concrete PHPUnit Version. They rely on features/behaviours of this concrete PHPUnit Version. If you use always the latest version, you'll soon have a lot of failing tests I think

Again, fully agree with you on this. If you can fix the build, and the difference in execution time is small, maybe we can end this nonsense?